src/conways.rs

```diff
@@ -0,0 +1,139 @@
+#[derive(Debug, Clone, PartialEq, Copy)]
+enum State {
+    Alive,
+    Dead,
+}
+
+type Position = (u32, u32);
+
+struct Grid {
+    cells: Vec<Vec<State>>,
+}
+
+impl Grid {
+    pub fn new(size: usize) -> Grid {
+        Grid {
+            cells: vec![vec![State::Dead; size]; size],
+        }
+    }
+
+    pub fn change_state(&mut self, position: Position, state: State) {
+        let x: usize = position.0 as usize;
+        let y: usize = position.1 as usize;
```

Comment:
You can deconstruct the tuple `(x,y) = position` inside the function or `(x,y):Position` in the function declaration